### DISCUSSION TITLE
I get no audio from browser when using youtube from the android.. any help?

### DISCUSSION THREAD

**Discussion by miaszh** [tesla-android/issue-tracker#263] [Q&A] [answered]
Hi
In the previous version 2023.7 I installed youtube on the Android, andwhen playing videos I had audio from the browser ..
Since 2023.18.1, there is no audio..
Even from the settings when tried to change the volume.. I got no audio..

Any help appreciated

**Comment by mikegapinski** [comment, marked as answer]
Reboot the car; it's mentioned on the settings page. From time to time the browser stops playing audio from any website 😅

**Reply by miaszh** [reply]
Will give it a try, thank you! :)

**Comment by ThatSmart** [comment]
> Hi
> 
> In the previous version 2023.7 I installed youtube on the Android, andwhen playing videos I had audio from the browser ..
> 
> Since 2023.18.1, there is no audio..
> 
> Even from the settings when tried to change the volume.. I got no audio..
> 
> 
> 
> Any help appreciated

Same issue here. Rebooting don't help.

**Reply by ThatSmart** [reply]
Turning the car off and on did the trick.

**Reply by mikegapinski** [reply]
Maybe one day the guys at Tesla will get it solved... This issue is there for a longer while already
